tap_circle_ci/schemas/deploy.json

```diff
@@ -0,0 +1,68 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
```

Comment:
Confirmed that this is a primary key, this cannot be null and please double check if this is string or number/integer?